README.md

```diff
@@ -39,14 +39,21 @@ Sign up to [get started for free](https://app.revenuecat.com/signup).
 🔀 | [Integrations](https://www.revenuecat.com/integrations) - over a dozen integrations to easily send purchase data where you need it
 💯 | Well maintained - [frequent releases](https://github.com/RevenueCat/purchases-ios/releases)
 📮 | Great support - [Help Center](https://community.revenuecat.com)
+🤩 | Awesome [new features](https://trello.com/b/RZRnWRbI/revenuecat-product-roadmap)  
 
 ## Getting Started
 For more detailed information, you can view our complete documentation at [docs.revenuecat.com](https://docs.revenuecat.com/docs).
 
-Or browse our iOS sample apps:
+Please follow the [Quickstart Guide](https://docs.revenuecat.com/docs/) for more information on how to install the SDK.
+
+Or view our iOS sample apps:
 - [MagicWeather](Examples/MagicWeather)
 - [MagicWeather SwiftUI](Examples/MagicWeatherSwiftUI)
 
+## Requirements
+- XCode 10.2+
```

Comment:
```suggestion
- Xcode 10.2+
```

🙊

Reply:
But we require Xcode 13.2 now.

Reply:
👍

Will that affect the minimum target (below)?

Reply:
Not directly, but that deployment target is outdated too. 

You can see our current deployment target specs here: 
https://github.com/RevenueCat/purchases-ios/blob/main/RevenueCat.podspec#L17

That means Xcode 13 + iOS 11.0+, tvOS 11.0+, macOS 10.13+, watchOS 6.2+

Reply:
Changed to Xcode to 13.2+ and min iOS to 11.0+

Reply:
Could we mention the whole group? i.e.: iOS, tvOS, macOS, watchOS with the targets I mentioned before

Reply:
Made a table for those platforms under Requirements